### PR TITLE
Prefer invulnerable over FORCE_CHANCE_TO_HIT

### PR DESCRIPTION
**Pull request by Slayer95**
This displays the chance as 0% in the battle GUI.

### CODE REVIEW

File: scenarios/01_Highest_Tiers.cfg [hunk 1]
```diff
@@ -292,7 +293,7 @@
             x=18
             y=40
             facing=se
-            placement,overwrite=map,yes
+            placement,overwrite,invulnerable=map,yes,yes
```
Comment (ProditorMagnus) [inline scenarios/01_Highest_Tiers.cfg]: dont like that you put unrelated attribute to same line

File: scenarios/17_Siege_of_Kharos.cfg [hunk 2]
```diff
@@ -856,7 +856,7 @@
                 {CLEAR_VARIABLE mastervar}
                 {MODIFY_UNIT id=Master moves 0}
                 {MODIFY_UNIT id=Master max_moves 0}
-                {FORCE_CHANCE_TO_HIT side_number=1,5,6 id=Master 0 ()}
+                {MODIFY_UNIT id=Master invulnerable yes}
```
Comment (ProditorMagnus) [inline scenarios/17_Siege_of_Kharos.cfg:859]: does it work?

Comment (Slayer95) [reply scenarios/17_Siege_of_Kharos.cfg:859]: it does

File: scenarios/01_Highest_Tiers.cfg (hunk 1 above)
Comment (Slayer95) [reply scenarios/01_Highest_Tiers.cfg]: fair

**Pull request merged**